lib/tasks/assets.rake

```diff
@@ -0,0 +1,18 @@
+namespace :assets do
```

Comment:
Is it worth duplicating these for whitehall assets too?  This approach works, but it means that the user will need to look up the asset by legacy URL path to get its ID first, in which case they may as well just delete it from the app console.

Reply:
I wanted to avoid having special cases for Whitehall because it's just more to delete when Whitehall eventually goes away. I think it if ends up being a big problem we could add them, but I wanted to experiment with not having them for now.

As it is, conveniently, the most common place this will get used in the docs you already have the ID of the asset easily available even if it's a whitehall asset because it recommends having an `asset` variable ready in the console.